joomla/joomla-cms
# [4.0] Remove "cardGrey" chrome

As written in https://github.com/joomla/joomla-cms/pull/30729, the "cardGrey" chrome is basically the same chrome as the "card" (once #30729 is merged) chrome. It just has an added "card-grey" module class.

However the same can be achieved by using the "module class" parameter in the module options and add "card-grey" there.

### Summary of Changes
* Removes the chrome layout /templates/cassiopeia/html/layouts/chromes/cardGrey.php
* Changes the styles of the positions which were using it to "card". That is "top-a", "main-top", "main-bottom" and "bottom-a".

### Testing Instructions
* Make sure to have the other PR applied as well, otherwise the chrome will not be found and you get some ugly display. Since it's RTC already, chances are high it's merged soon.
* Add the class "card-grey" to the module class parameter in your module

### Actual result BEFORE applying this Pull Request
No change as the class is there anyway, now twice.


### Expected result AFTER applying this Pull Request
Position gets the darker background only if that class is added to the module, otherwise the regular white background is used.


### Documentation Changes Required
None
